app/datatables/volunteer_datatable.rb

```diff
@@ -28,17 +30,20 @@ def data
           occurred_at: I18n.l(volunteer.most_recent_attempt_occurred_at, format: :full, default: nil)
         },
         supervisor: {id: volunteer.supervisor_id, name: volunteer.supervisor_name},
-        hours_spent_in_days: volunteer.hours_spent_in_days(30)
+        hours_spent_in_days: volunteer.hours_spent_in_days(30),
+        extra_languages: volunteer.languages&.map { |lang| {id: lang.id, name: lang.name} }
       }
     end
   end
 
   def filtered_records
-    raw_records
-      .where(supervisor_filter)
-      .where(active_filter)
-      .where(transition_aged_youth_filter)
-      .where(search_filter)
+    extra_languages_filter do
```

Comment:
naming a variable "extra" seems unclear and not ideal :)

Reply:
I'll rename it too :)